README.md

```diff
@@ -1,32 +1,190 @@
+  
+![new database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newdb.png)
+    
+5. Name the new databases "pets" and press Save.    
+     
+![name pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/namepetsdb.png)
+    
+6. Confirm the new database named `pets` appears in the server view:    
+     
+![new pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newpetdb.png)
+
+
+We will use the **Query Tool** to execute SQL statements.
+
+1. Click on the `pets` database to select the database.
+2. Click on the `query tool` icon.
+ 
+![open query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/openquerytool.png)
+
+The query tool opens a panel where we can write SQL statements.
+Confirm the connection is for  "pets/postgres@PostgreSQL 15". 
+
+![query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/querytool.png)
+
+
+
+## CREATE TABLE
+
+The `CREATE TABLE` statement creates a new table
+by specifying the table name and the column details.
+Each column is assigned a name and data type, along with other optional attributes.
+
+While PostgreSQL supports dozens of data types, we will primarily work with the following:
+
+- INTEGER stores integer values
+- DECIMAL stores fractional numeric values
+- TEXT stores textual values
+- DATE stores year, month, and day
+
+#### Coding Conventions
+
+1. Use uppercase for SQL keywords such as CREATE, TABLE, INTEGER, PRIMARY KEY, etc.
+   SQL is case insensitive, but it is a convention
+   to uppercase SQL keywords to separate them from table names and columns.
+2. Use lowercase letters for table and column names such as `pet`, `species`, etc.
+3. When we have multiple words in a table or column name,
+   link them together using underscores rather than spaces or camelcase.
+
+The `CREATE TABLE` statement for the `pet` table is shown below:
+
+```code
+CREATE TABLE pet (
+	id  INTEGER PRIMARY KEY,
+	name TEXT NOT NULL,
+	species TEXT NOT NULL,
+	breed TEXT,
+	age INTEGER
+);
+```
+
+Let's break down the above code:
+
+1. The `CREATE TABLE` statement specifies the table name `pet`.
+2. The `pet` table has 5 columns:
+   - `id` will hold a unique integer value to reference each pet.
+     - every table should have a primary key to ensure each row is unique.
+   - `name` and `species` are required fields that can't be null.
+   - `breed` and `age` allow null values as the breed and age of a pet may be unknown.
+
+We will use the query tool to execute the CREATE statement:
+
+1. Enter the `CREATE TABLE` statement into the query panel.
+2. Press the execute button.
+3. A message should display indicating the query was successful.
+
+![create table](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/createtable.png)
+
+
+We can confirm the table was added to the `pets` database by expanding the database
+tree structure as shown below:
```

Comment:
Potential place to explain/define the word "schema"?

Reply:
good idea.  updated with a brief explanation of schema.